sdk/aqueduct/backend/api_client.py

```diff
@@ -254,10 +254,10 @@ def list_tables(self, integration_id: str) -> List[str]:
         return [x for x in resp.json()["object_names"]]
 
     def connect_integration(
-        self, name: str, service: ServiceType, config: IntegrationConfig
+        self, name: str, service: Union[str, ServiceType], config: IntegrationConfig
     ) -> None:
         integration_service = service
-        if not isinstance(integration_service, str):
```

Comment:
@kenxu95 I had to change this because `if not isinstance(ServiceType.K8S, str):` actually evaluates to False, so this line wasn't working before...

Reply:
Oh right cus the enum inherits from string...